smithy-aws-protocol-tests/model/restJson1/malformedRequests/malformed-set.smithy

```diff
@@ -34,6 +34,117 @@ apply MalformedSet @httpMalformedRequestTests([
             }
         }
     },
+    {
+        id: "RestJsonMalformedSetDuplicateDocuments",
+        documentation: """
+        When the set has duplicated documents, the response should be a 400
```

Comment:
Why not indent long strings to make it easier to see key value pairs?

Reply:
I copy and pasted the same mistake as the test above